client/package-lock.json

```diff
@@ -49,15 +49,14 @@
       }
     },
     "@babel/generator": {
-      "version": "7.6.0",
```

Comment:
`package-lock.json` should not be modified either (except the parts related to react-s-alert). Please change it.